deploy/csv-templates/ics-operator.csv.yaml.in

```diff
@@ -382,6 +427,7 @@ spec:
           - patch
           - update
           - watch
+          - watch
```

Comment:
`watch` is duplicated in several places here

Reply:
Thanks for catching it,

Tried to update all the places, please have a look once again